### PR TITLE
Fix some phpdoc

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix some phpdoc
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No need for QA

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20092)

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ public function overviewAction()
      *
      * Parse all errors mapped by id html field
      *
-     * @param Form $form The form
+     * @param Form $form
```
Comment (matks) [inline src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php:90]: Reminded me of

File: src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php [hunk 2]
```diff
@@ -249,7 +249,7 @@ protected function getDemoModeErrorMessage()
     /**
      * Checks if the attributes are granted against the current authentication token and optionally supplied object.
      *
-     * @param string $controller name of the controller to valide access
+     * @param string $controller name of the controller that token is tested against
```
Comment (matks) [inline src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php:252]: Frenglish

File: src/PrestaShopBundle/Controller/Api/ApiController.php [hunk 3]
```diff
@@ -194,8 +197,8 @@ protected function jsonResponse(
     /**
      * Checks if access is granted.
      *
-     * @param string $controller name of the controller
      * @param array $accessLevel
+     * @param string $controller name of the controller
```
Comment (matks) [inline src/PrestaShopBundle/Controller/Api/ApiController.php:201]: 🤦

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**